core/src/main/java/org/elasticsearch/index/seqno/GlobalCheckpointSyncAction.java

```diff
@@ -43,8 +43,8 @@
 
 import java.io.IOException;
 
-public class GlobalCheckpointSyncAction extends TransportReplicationAction<GlobalCheckpointSyncAction.PrimaryRequest,
-    GlobalCheckpointSyncAction.ReplicaRequest, ReplicationResponse> {
+public class GlobalCheckpointSyncAction extends TransportReplicationAction<GlobalCheckpointSyncAction.GlobalCheckpointPrimaryRequest,
+    GlobalCheckpointSyncAction.GlobalCheckpointReplicaRequest, ReplicationResponse> {
```

Comment:
why rename this , it's already scoped under GlobalCheckpointSyncAction. it gets so long...